lib/cmd/common/version.js

```diff
@@ -41,9 +49,11 @@ module.exports = {
       cachedLatest = JSON.parse(cachedLatest)
     } catch (error) {}
     if (!_.isEmpty(cachedLatest)) {
-      if (cachedLatest.ts - (new Date().getTime() - (dayInMs * daysToCache)) > 0) {
-        self.latest = cachedLatest;
-        return cb(cachedLatest.is);
+      if(cachedLatest.current && semver.eq(cleanupVersion(cachedLatest.current), cleanupVersion(fhc._version))){
```

Comment:
it's very weird that we're caching our version.
Seems to be due to us caching whether we are up to date, rather than the version number available on the upstream server.